src/agent/src/config.rs

```diff
@@ -26,6 +26,7 @@ const LOG_VPORT_OPTION: &str = "agent.log_vport";
 const CONTAINER_PIPE_SIZE_OPTION: &str = "agent.container_pipe_size";
 const UNIFIED_CGROUP_HIERARCHY_OPTION: &str = "agent.unified_cgroup_hierarchy";
 const CONFIG_FILE: &str = "agent.config_file";
+const REST_API_OPTION: &str = "agent.guest_components_rest_api";
```

Comment:
Would it improve clarity to align `REST_API_OPTION` with its actual value? For instance, changing `REST_API_OPTION` to `GUEST_COMPONENTS_REST_API_OPTION` could enhance consistency.